todo_app/.env.test

```diff
@@ -1,11 +1,11 @@
 # Flask server configuration.
```

Comment:
Your tests are not using this file when running in the docker test container, right? Because you haven't passed it in. Instead, you're running all your tests with those encrypted env vars. Can you fix that?

Reply:
I think my tests are still using this file. This is handled by the `conftest.py` file in integration tests. For unit tests it does not matter as none of the envs are used.